### PR TITLE
(chore, project management) Pull Request Template

### PR DESCRIPTION
**Pull request by kuhnchris**
## Check List

* [X] I understood and followed the [Contribution guide](https://github.com/revoltchat/revolt/discussions/282)
* [n/a] I tested my change locally and it is working as intended
* [X] The change doesn't have any notable side effects on other project
* [n/a] (optional) I opened a pull request on the [translation repository](https://github.com/revoltchat/translations)
* [n/a] I added screenshots, so the testing can be done quicker and easier.

---
As proposed over in #488

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Please make sure to check the following tasks before opening and submitting a PR
+
+* [ ] I understood and followed the [Contribution guide](https://github.com/revoltchat/revolt/discussions/282)
+* [ ] I tested my change locally and it is working as intended
+* [ ] The change doesn't have any notable side effects on other project
+* [ ] (optional) I opened a pull request on the [translation repository](https://github.com/revoltchat/translations)
+* [ ] I added screenshots, so the testing can be done quicker and easier.
```
Comment (Rexogamer) [inline .github/pull_request_template.md:7]: I feel like this should be reworded as such (or similar):
```suggestion
* [ ] I have included screenshots to demonstrate my changes
```

**Review by Rexogamer — CHANGES_REQUESTED** [review]
Mostly LGTM aside from a few nitpicks, but it seems as though you've accidentally included your changes in #488?

**Comment by kuhnchris** [comment]
> Mostly LGTM aside from two minor nitpicks, but it seems as though you've accidentally included #488's changes?

Thanks for checking, will add your changes ASAP -  and yes, I accidently commited on master on #488, not entirely sure how to separate this change out of that without breaking #488... Maybe you got an idea?

**Comment by Rexogamer** [comment]
You might have to close this, delete the current `pr_template` branch, checkout master then re-branch it using `git branch pr_template e13edcbb760c9d68b9eb1d4b59de7a58d8d8f16f`

**Pull request closed without merging**